doc/source/ray-core/ray-dashboard.rst

```diff
@@ -195,28 +413,33 @@ Below is an example with a `traefik <https://doc.traefik.io/traefik/getting-star
         [[http.services.dashboard.loadBalancer.servers]]
           url = "http://localhost:8265"
 
-References
-----------
+How to disable the dashboard
```

Comment:
Same as above, we should either show how to do it in both vm and kuberay here or send users to the dashboard section under the ray cluster documentation where we list all different ways to do it.

Reply:
done. But idk how to do it in k8s. do you know who I should follow up with?

Reply:
cc: @kevin85421 or @architkulkarni ?